### DISCUSSION TITLE
Curious about the next release

### DISCUSSION THREAD

**Discussion by Roe-bit** [tiltedphoques/TiltedEvolution#542] [Q&A]
I'm excited to see a lot of stuff get fixed based on how many email updates I get pertaining to this repo.
Is there a changelog we can see for what comes next, maybe a projected release date for 1.3.3 or 1.4?

**Comment by absol89** [comment]
Version 1.7 is being scanned for approval on Nexus with this changelog: https://github.com/tiltedphoques/TiltedEvolution/pull/742
